### PR TITLE
Add Natal as a site for the Mar 2024 Hackathon

### PR DESCRIPTION
**Pull request by mribeirodantas**

### CODE REVIEW

File: src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Hackathon - March 2024 (Natal)
+subtitle: Local node of the nf-core hackathon at Instituto do Cérebro, UFRN, Natal - Brazil
+type: hackathon
+start_date: '2024-03-18'
+start_time: '10:00+02:00'
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md:6]: Are you sure about the time zone here?

File: src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+end_date: '2024-03-20'
+end_time: '18:00+02:00'
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md:8]: same

File: src/content/events/2024/hackathon-march-2024/index.md [hunk 3]
```diff
@@ -80,6 +80,7 @@ When site registration closes, we will open participant registration with your s
 | 🇵🇱 Poland  | Kraków     | Ardigen                                 | [Read more](./hackathon-march-2024/krakow-ardigen.md)           |
 | 🇪🇸 Spain   | Barcelona  | CRG                                     | [Read more](./hackathon-march-2024/bcn-crg.md)                  |
 | 🇸🇪 Sweden  | Stockholm  | SciLifeLab                              | [Read more](./hackathon-march-2024/sweden-scilifelab.md)        |
+| 🇧🇷 Brazil  | Natal      | Instituto do Cérebro - UFRN             | [Read more](./hackathon-march-2024/brazil-ice-ufrn.md)          |
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/index.md:83]: Please sort this table alphabetically

**Review by mashehu — APPROVED** [review]

File: src/content/events/2024/hackathon-march-2024/index.md [hunk 4]
```diff
@@ -70,17 +70,17 @@ When site registration closes, we will open participant registration with your s
...
+| 🇩🇰 Denmark | Copenhagen | ZS Associates                           | [Read more](./hackathon-march-2024/denmark-zs.md)               |
+| 🇫🇷 France  | Rennes     | IGDR                                    | [Read more](./hackathon-march-2024/france-igdr.md)              |
+| 🇩🇪 Germany | Heidelberg | GHGA                                    | [Read more](./hackathon-march-2024/germany-heidelberg.md)       |
+| 🇮🇹 Italy   | Pavia      | Dept. Biology and Biotechnology - UniPV | [Read more](./hackathon-march-2024/italy-unipv.md)              |
+| 🇵🇱 Poland  | Kraków     | Ardigen                                 | [Read more](./hackathon-march-2024/krakow-ardigen.md)           |
+| 🇪🇸 Spain   | Barcelona  | CRG                                     | [Read more](./hackathon-march-2024/bcn-crg.md)                  |
+| 🇸🇪 Sweden  | Stockholm  | SciLifeLab                              | [Read more](./hackathon-march-2024/sweden-scilifelab.md)        |
```
Comment (mribeirodantas) [inline src/content/events/2024/hackathon-march-2024/index.md:83]:
```suggestion
| 🇸🇪 Sweden  | Stockholm  | SciLifeLab                              | [Read more](./hackathon-march-2024/sweden-scilifelab.md)        |
| 🇬🇧 United Kingdom | London     | Cosyne Therapeutics                     | [Read more](./hackathon-march-2024/UK-London.md)                |
```

File: src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+address: R. do Horto - Lagoa Nova, Natal - RN, 59076-550
+location_url: https://neuro.ufrn.br/
+location_latlng: [-5.833473893839016, -35.20483247479188]
+---
+
+Local event to be held at Instituto do Cérebro, UFRN.
+
+Contact:
+
+- [<i class="fab fa-slack"></i> João Victor Cavalcante](https://nfcore.slack.com/team/U044E439ABX)
```
Comment (jvfe) [inline src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md:19]:
```suggestion
- [<i class="fab fa-slack"></i> João Vitor Cavalcante](https://nfcore.slack.com/team/U044E439ABX)
```

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md (hunk 5 above)
Comment (mribeirodantas) [reply src/content/events/2024/hackathon-march-2024/brazil-ice-ufrn.md:19]: Sorry 😬

**Pull request merged**